backend/src/main/scala/com/softwaremill/bootzooka/api/Routes.scala

```diff
@@ -28,20 +46,12 @@ trait Routes extends UsersRoutes
     } & handleRejections(rejectionHandler)
   }
 
-  val routes =
+  def base(route: Route) =
```

Comment:
Right, maybe the name isn't the best one then, but unfortunately I don't have many other ideas ;)
If it was a directive, maybe it could simple be a `requestWrapper`? I think all those directives could be combined using `&`, into a single one, no?